diff -rupN dwm-6.0/config.def.h dwm-6.0p/config.def.h
--- dwm-6.0/config.def.h	2013-02-21 13:48:08.477464918 -0800
+++ dwm-6.0p/config.def.h	2013-02-21 15:18:57.015502677 -0800
@@ -47,7 +47,7 @@ static const Layout layouts[] = {
 
 /* commands */
 static const char *dmenucmd[] = { "dmenu_run", "-fn", font, "-nb", normbgcolor, "-nf", normfgcolor, "-sb", selbgcolor, "-sf", selfgcolor, NULL };
-static const char *termcmd[]  = { "uxterm", NULL };
+static const char *termcmd[]  = { "st", NULL };
 
 static Key keys[] = {
 	/* modifier                     key        function        argument */
